FE/src/components/Admin.jsx

```diff
@@ -1,8 +1,301 @@
+                    setrecipeTags(false);
+                }}>All Users</button>
+                <button onClick={() => {
+                    setAllUsers(false);
+                    setReviewedRecipes(true);
+                    setrecipeTags(false);
+                }}>Reviewed Recipes</button>
+                <button onClick={() => {
+                    setAllUsers(false);
+                    setReviewedRecipes(false);
+                    setrecipeTags(true);
+                }}>Recipe Tags</button>
+            </div>
+            {error && <p>{error}</p>}
+            {fail && <div className='fail'><h2>Please Sign In</h2><br/> <button type='login' onClick={signIn}>Log In</button></div>}
+            {success &&
+            <div>
+                {allUsers &&
+                <div className = 'allUsers'>
+                    <h2>All Users</h2>
+                    {usersList.map((user)=> {
+                        return (
+                            <div className ="userCard" key = {user.id}>
+                            <img src={user.imgurl} alt={`User account image for ${user.username}`} height="15%" width="22.5%"/>
+                            <h3>Username: {user.username}</h3>
+                            <p>Name: {user.name}</p>
+                            <p>Email: {user.email}</p>
+                            <p>Review count: {user.reviewcount}</p>
+                            <button onClick={() => {
+                                navigate(`/users/${user.id}`)
+                            }}>View User</button>
+                        </div>
+                        )
+                    })}
+                </div>}
+                {reviewedRecipes &&
+                <div className = 'reviewedRecipes'>
+                    <h2>All Reviewed Recipes</h2>
+                    {reviewedRecipesList.map((reviewedRecipe)=> {
+                        return (
+                            <div className ="reviewedRecipeCard" key = {reviewedRecipe.id}>
+                                <h3>{reviewedRecipe.title}</h3>
+                                <h4>{reviewedRecipe.user.username}</h4>
+                                <img src={reviewedRecipe.imgurl} alt={`A picture of ${reviewedRecipe.title}`} height="15%" width="22.5%"/>
+                                {reviewedRecipe.tags.map((tag)=> {
+                                    return (
+                                        <div className ='cardTag' key = {tag.id}>
+                                            <p><em>{tag.name}</em></p>
+                                        </div>
+                                    )
+                                })}
+                                {reviewedRecipe.reviews.map((review)=> {
+                                    return (
+                                        <div className ='cardReview' key = {review.id}>
+                                            <p>Review: "{review.content}" by {review.user.username}</p>
```

Comment:
Do we want this? I didn't structure the data to include child reviews because I thought it might clutter up the page. Can we just have them click on the SingleRecipe page instead?

Reply:
wasn't sure if we'd want a drop down later something for easy admin review? Again i'm still aiming for tier 2 features eventually so maybe its a wit till after MVP?